tycho-indexer/src/extractor/dynamic_contract_indexer/hook_orchestrator.rs

```diff
@@ -76,6 +82,7 @@ impl HookOrchestratorRegistry {
             .insert(hook_address, orchestrator);
     }
 
+    #[allow(dead_code)]
```

Comment:
allowing dead_code here because, by design, a custom orchestrator might be needed in the future for custom integrations